test/sharness/t0110-gateway.sh

```diff
@@ -35,13 +35,14 @@ test_expect_success "GET IPFS path output looks good" '
 
 test_expect_success "GET IPFS directory path succeeds" '
   mkdir dir &&
-  echo "12345" > dir/test &&
-  HASH2=`ipfs add -r -q dir | tail -n 1` &&
-  wget "http://127.0.0.1:$port/ipfs/$HASH2"
+  echo "12345" >dir/test &&
+  ipfs add -r -q dir >actual &&
+  HASH2=$(tail -n 1 actual) &&
```

Comment:
curious, is there's a reason other than readability for switching into two lines (using `actual`)? (there may be some side effect i'm not seeing)